arches_lingo/src/arches_lingo/components/widgets/ConceptResourceSelectWidget/components/ConceptResourceSelectWidgetEditor.vue

```diff
@@ -0,0 +1,304 @@
+<script setup lang="ts">
+import arches from "arches";
+import { computed, ref, useTemplateRef } from "vue";
+
+import { useGettext } from "vue3-gettext";
+import { FormField } from "@primevue/forms";
+
+import Button from "primevue/button";
+import Message from "primevue/message";
+import MultiSelect from "primevue/multiselect";
+
+import { fetchConceptResources } from "@/arches_lingo/api.ts";
+import { getItemLabel } from "@/arches_vue_utils/utils.ts";
```

Comment:
We archived arches_vue_utils and merged it into component lab yesterday, so you'll want to merge the latest and replace these.